sklearn/utils/tests/test_validation.py

```diff
@@ -539,3 +540,29 @@ def test_suppress_validation():
     assert_all_finite(X)
     sklearn.set_config(assume_finite=False)
     assert_raises(ValueError, assert_all_finite, X)
+
+
+class DummyMemory(object):
+    def __init__(self):
+        pass
+
+    def cache(self, func):
+        return func
+
+    cachedir = None
```

Comment:
oh ok.

I only wanted to make test something like 

```python
memory = check_memory(DummyMemory())
assert_equal(memory.cachedir = 'something')
```

but it can be cachedir or anything else as attributes